lite2/client.go

```diff
@@ -714,39 +714,52 @@ func (c *Client) bisection(
 	newVals *types.ValidatorSet,
 	now time.Time) error {
 
+	type HeaderSet struct {
```

Comment:
any particular reason this is exposed?

Reply:
didn't realise that a struct declared in a function was exposed - I thought it was by default only available locally within that function